tests/integration-tests-topologies/src/main/java/org/apache/pulsar/tests/topologies/PulsarCluster.java

```diff
@@ -193,24 +208,25 @@ public void start() throws Exception {
             String name = "pulsar-" + serviceName + "-" + i;
             T container = containerCreator.apply(name);
             containers.put(name, container);
-            startFutures.add(CompletableFuture.runAsync(() -> container.start()));
         }
-        CompletableFuture.allOf(startFutures.toArray(new CompletableFuture[startFutures.size()])).join();
-        log.info("Successfully started {} {} containers", numContainers, serviceName);
         return containers;
     }
 
     public void stop() {
-        Stream.of(proxyContainer, csContainer, zkContainer).parallel().forEach(GenericContainer::stop);
-        workerContainers.values().parallelStream().forEach(GenericContainer::stop);
-        brokerContainers.values().parallelStream().forEach(GenericContainer::stop);
-        bookieContainers.values().parallelStream().forEach(GenericContainer::stop);
-        if (null != spec.externalServices()) {
-            spec.externalServices().values()
-                .parallelStream()
-                .forEach(GenericContainer::stop);
+
+        Stream<GenericContainer> containers = Streams.concat(
```

Comment:
again, don't mix changing styles in this PR, unless there is a strong reason.

Reply:
it's meant to parallelize all container shutdown.